docs/import/manage_disaster.js

```diff
@@ -421,7 +423,7 @@ function getCreateFolderPromise(dir) {
   return new Promise(
       (resolve, reject) =>
           ee.data.createFolder(dir, false, (result, failure) => {
-            if (failure) {
+            if (failure && !failure.startsWith('Cannot overwrite asset ')) {
```

Comment:
why only for these failures?

Reply:
The "cannot overwrite asset" failure is benign, because it means the folder is already there, which is what we wanted. Other failures might not be ok (permissions errors, etc.), so we throw. Is that your question?